examples/leasing/leasing

```diff
@@ -0,0 +1,6 @@
+0 lease_test 1
+1 lease_test 2
+2 free_pool 0
```

Comment:
Would you document the format of this file? Not immedately obvious what each of the columns means.